### PR TITLE
Mario/create view test

### PR DESCRIPTION
**Pull request by ElTragon**

### CODE REVIEW

File: cypress/integration/dolthub/productionTests/privatePaths/render/newDatabaseWithEdits/other.spec.ts [hunk 1]
```diff
@@ -1,3 +1,4 @@
+import { testViewQuery } from "cypress/integration/utils/sharedTests/testViewQuery";
```
Comment (tbantle22) [inline cypress/integration/dolthub/productionTests/privatePaths/render/newDatabaseWithEdits/other.spec.ts:1]: Make sure you use the relative path

File: cypress/integration/dolthub/productionTests/publicPaths/render/database/corona-virus.spec.ts [hunk 2]
```diff
@@ -61,6 +67,18 @@ describe(`${pageName} renders expected components on different devices`, () => {
       newShouldArgs("be.visible.and.contain", ["No authentication", "sign in"]),
     ),
     testViewsSection(hasBranch, 15, testView),
+    newExpectationWithClickFlows(
+      "should show view query button",
```
Comment (tbantle22) [inline cypress/integration/dolthub/productionTests/publicPaths/render/database/corona-virus.spec.ts:71]:
```suggestion
      "should show create view button",
```

File: cypress/integration/utils/sharedTests/testViewQuery.ts [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+  newExpectationWithClickFlows(
+    "should be able to navigate to database tab",
+    "[data-cy=repo-database-tab]",
+    beVisible,
+    [newClickFlow("[data-cy=repo-database-tab]", [])],
+  ),
+
+  // SAVE THE QUERY
+  newExpectationWithClickFlows(
+    "should show view query button",
```
Comment (tbantle22) [inline cypress/integration/utils/sharedTests/testViewQuery.ts:26]:
```suggestion
    "should show create view button",
```

File: cypress/integration/utils/sharedTests/testViewQuery.ts [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+  newExpectationWithClickFlows,
+} from "../helpers";
+import { Tests } from "../types";
+import {
+  beVisible,
+  beVisibleAndContain,
+  typingExpectation,
+} from "./sharedFunctionsAndVariables";
+
+const testQueryName = "test-view-name";
```
Comment (tbantle22) [inline cypress/integration/utils/sharedTests/testViewQuery.ts:13]:
```suggestion
const testViewName = "test-view-name";
```

File: cypress/integration/utils/sharedTests/testViewQuery.ts [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+    [newClickFlow("[data-cy=create-commit]", [])],
+  ),
+  newExpectationWithClickFlows(
+    "should be able to create commit",
+    "[data-cy=create-commit-button]",
+    beVisible,
+    [newClickFlow("[data-cy=create-commit-button]", [])],
+  ),
+  newExpectationWithClickFlows(
+    "the saved query should be rendered in the queries tab",
```
Comment (tbantle22) [inline cypress/integration/utils/sharedTests/testViewQuery.ts:55]:
```suggestion
    "the view should be rendered in the views tab",
```

File: cypress/integration/utils/sharedTests/testViewQuery.ts [hunk 6]
```diff
@@ -0,0 +1,83 @@
...
+          "should have saved view listed",
+          `[data-cy=repo-view-views-${testQueryName}`,
+          beVisibleAndContain(testQueryName),
+        ),
+      ]),
+    ],
+  ),
+
+  newExpectationWithClickFlows(
+    "the saved query description should be rendered",
```
Comment (tbantle22) [inline cypress/integration/utils/sharedTests/testViewQuery.ts:70]: Don't think views have a description?

**Review by liuliu-dev — APPROVED** [review]

File: cypress/integration/dolthub/productionTests/publicPaths/render/database/corona-virus.spec.ts (hunk 2 above)
Comment (ElTragon) [reply cypress/integration/dolthub/productionTests/publicPaths/render/database/corona-virus.spec.ts:71]: Done

File: cypress/integration/utils/sharedTests/testViewQuery.ts (hunk 4 above)
Comment (ElTragon) [reply cypress/integration/utils/sharedTests/testViewQuery.ts:13]: Done

File: cypress/integration/utils/sharedTests/testViewQuery.ts [hunk 7]
```diff
@@ -0,0 +1,83 @@
...
+export const testViewQuery: Tests = [
+  // NAVIGATE TO THE DATABASE TAB
+  newExpectationWithClickFlows(
+    "should be able to navigate to database tab",
+    "[data-cy=repo-database-tab]",
+    beVisible,
+    [newClickFlow("[data-cy=repo-database-tab]", [])],
+  ),
+
+  // SAVE THE QUERY
```
Comment (tbantle22) [inline cypress/integration/utils/sharedTests/testViewQuery.ts:24]: Update

File: cypress/integration/utils/sharedTests/testViewQuery.ts (hunk 5 above)
Comment (tbantle22) [reply cypress/integration/utils/sharedTests/testViewQuery.ts:55]: You missed this one

**Review by tbantle22 — APPROVED** [review]

**Pull request merged**